packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/api/IGraphicalChangesEvent.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.api;
+
+/**
+ * Use to create event record to be handled by IRepresentationEventHandler when doing an undo/redo.
```

Comment:
You are making tons of brittle links between concepts in the Javadoc. Things will change and quickly for such new code. Detail more the role of the concept instead of an assumption on which other concept will use it. This will change even during the review.